OpenMTRDemo/Forms/TiledFiltersForm.cs

```diff
@@ -0,0 +1,391 @@
+        private void pictureBoxPane4_DoubleClick(object sender, EventArgs e)
+        {
+            try
+            {
+                ExpandedImageForm expandedImageForm;
+                if (meterPane1.ModifiedImage == meterPane4.SourceImage)
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane4.SourceImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+                else
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane4.ModifiedImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+
+            }
+            catch (Exception)
+            {
+
+            }
+        }
+
+        private void pictureBoxPane5_DoubleClick(object sender, EventArgs e)
+        {
+            try
+            {
+                ExpandedImageForm expandedImageForm;
+                if (meterPane1.ModifiedImage == meterPane5.SourceImage)
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane5.SourceImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+                else
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane5.ModifiedImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+
+            }
+            catch (Exception)
+            {
+
+            }
+        }
+
+        private void pictureBoxPane6_DoubleClick(object sender, EventArgs e)
+        {
+            try
+            {
+                ExpandedImageForm expandedImageForm;
+                if (meterPane1.ModifiedImage == meterPane6.SourceImage)
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane6.SourceImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+                else
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane6.ModifiedImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+            }
+            catch (Exception)
+            {
+
+            }
+        }
+
+        private void checkBoxGuassianBlur_CheckedChanged(object sender, EventArgs e)
+        {
+            try
+            {
+                KeyValPair keyValuePair = new KeyValPair();
+                keyValuePair = getMeterKeyValuePair();
+                if (checkBoxGuassianBlur.Checked)
+                {
+                    
+                    Cv2.GaussianBlur(keyValuePair.meter.SourceImage, keyValuePair.meter.ModifiedImage, new OpenCvSharp.Size((int)numericUpDown1.Value, (int)numericUpDown2.Value), 0, 1, BorderTypes.Default);
```

Comment:
I would recommend extracting `new OpenCvSharp.Size((int)numericUpDown1.Value, (int)numericUpDown2.Value)` to a separate variable for readability.